src/tes/worker/engine.go

```diff
@@ -77,7 +78,9 @@ func (eng *engine) RunJob(parentCtx context.Context, jobR *pbr.JobResponse) erro
 	}()
 
 	// Ticker for State polling
-	tickChan := time.NewTicker(time.Second * 5).C
+	pollRate := time.Duration(eng.conf.StatusPollRate)
+	tickChan := time.NewTicker(pollRate * time.Millisecond).C
```

Comment:
Making the config struct have type of `time.Duration` would mean we don't need these lines to convert it from an int.